<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tw_page">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>391</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string> 登录  </string>
    </attribute>
    <widget class="QPushButton" name="pb_commit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>330</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_passwd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1881888188</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_passwd">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>123</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string> 注册 </string>
    </attribute>
    <widget class="QLineEdit" name="le_passwd_register">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入20位以内密码&lt;/p&gt;&lt;p&gt;可以是字符或数字&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_passwd_register">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tel_register">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel_register">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入8-11位手机号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear_register">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit_register">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>330</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_name__register">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name_register">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入十位以内名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_confirm_passwd">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;确认密码，保证两次输入一致&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_confirm_passwd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/images/register.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_tel_register</tabstop>
  <tabstop>le_passwd_register</tabstop>
  <tabstop>le_confirm_passwd</tabstop>
  <tabstop>le_name_register</tabstop>
  <tabstop>pb_clear_register</tabstop>
  <tabstop>pb_commit_register</tabstop>
  <tabstop>tw_page</tabstop>
  <tabstop>le_tel</tabstop>
  <tabstop>le_passwd</tabstop>
  <tabstop>pb_commit</tabstop>
  <tabstop>pb_clear</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
